### DISCUSSION TITLE
Broken website or conflicting dark mode?

### DISCUSSION THREAD

**Discussion by hjpoe** [darkreader/darkreader#7701] [Q&A]
I believe I have the same issue as #7689 except with the Stack Overflow website and the GitHub website. The page first shows up as all black and white, and then goes into the normal mode (I use Dynamic theme in Dark Reader). 

### System Information:
Windows 10 - dark mode enabled
Chrome 96.0.4664.110 - dark theme enabled

I also have dark theme enabled in Stack Overflow. I just checked whether it still happens with light mode enabled on the website, same thing happened.

On GitHub, I have the dimmed dark theme enabled. It happens on all of the other themes as well. (I'm switching to light theme with Dark Reader now, it looks nicest.)

Is this just another couple of broken websites, or is this an issue with some conflicting dark modes?

**Comment by Gusted** [comment]
> The page first shows up as all black and white, and then goes into the normal mode (I use Dynamic theme in Dark Reader).

Could you explain a bit on this one. In what kind of state is the site when it's black and white (and is this the site's theme?) and what is this normal mode? I don't know what you're trying to mean by that.

Regards,
Gusted
